### PR TITLE
[Feature][EditText] Add props of EditText

### PR DESCRIPTION
**Pull request by jinsunee**
## Description

I added props about Border and Icon type of Element(props for `box` type input).
and also fixed test code according to this changed.
 
- About border
 `borderStyle`, `borderWidth` for customizing border style(height or border width).

- About Icon (Element)
 `leftElement`, `leftElementStyle`, `rightElement`, `rightElementStyle`

[README.md](https://github.com/ilikeu7246/dooboo-ui-native/blob/feat/edit-text-add-props/src/components/shared/EditText/README.md) related to this PR is here.
And this is preview image of feature i implemented.

| | Default | Row |
|--|---------------|--------------|
| `underlined` |underlined_default | underlined_row |
| <center>`box`</center> |default | row |

## Checklist

- [x] I added props about border. [`borderStyle`, `borderWidth`]
- [x] I added props about Icon. [`leftElement`, `leftElementStyle`, `rightElement`, `rightElementStyle`]
- [x] I updated README.md of this component.
- [x] I fixed test code

### CODE REVIEW

File: src/components/shared/EditText/index.tsx [hunk 1]
```diff
@@ -114,6 +130,10 @@ interface Props {
   onSubmitEditing?: (
     e: NativeSyntheticEvent<TextInputSubmitEditingEventData>,
   ) => void;
+  leftIcon?: ReactElement;
```
Comment (hyochan) [inline src/components/shared/EditText/index.tsx]: leftElement?

File: src/components/shared/EditText/index.tsx [hunk 2]
```diff
@@ -114,6 +130,10 @@ interface Props {
...
+  leftIconStyle?: ViewStyle;
```
Comment (hyochan) [inline src/components/shared/EditText/index.tsx]: leftElementStyle?

File: src/components/shared/EditText/index.tsx [hunk 3]
```diff
@@ -114,6 +130,10 @@ interface Props {
...
+  rightIcon?: ReactElement;
```
Comment (hyochan) [inline src/components/shared/EditText/index.tsx]: rightElement?

File: src/components/shared/EditText/index.tsx [hunk 4]
```diff
@@ -114,6 +130,10 @@ interface Props {
...
+  rightIconStyle?: ViewStyle;
```
Comment (hyochan) [inline src/components/shared/EditText/index.tsx]: rightElementStyle?

File: src/components/shared/EditText/index.tsx [hunk 5]
```diff
@@ -127,27 +148,39 @@ interface Props {
...
+}
+
 function EditText(props: Props): ReactElement {
   const [focused, setFocus] = useState(false);
 
   const {
     testID,
     errorTestID,
-    isRow = false,
+    type = EditTextInputType.DEFAULT,
```
Comment (hyochan) [inline src/components/shared/EditText/index.tsx:164]: 👍

File: src/components/shared/EditText/index.tsx [hunk 6]
```diff
@@ -160,143 +193,281 @@ function EditText(props: Props): ReactElement {
...
-              : { borderWidth: 0.6, borderRadius: inputContainerRadius },
-          ]}
-        >
-          {label ? (
-            <StyledRowLabel
-              style={[
-                labelTextStyle,
-                errorText
+  switch (type) {
+    case EditTextInputType.DEFAULT:
```
Comment (hyochan) [inline src/components/shared/EditText/index.tsx:197]: Try to make this case as `default:`. Not putting `default` condition may result in returning `undefined`.

Comment (jinsunee) [reply src/components/shared/EditText/index.tsx:197]: Thank you!!😆

File: src/components/shared/EditText/README.md [hunk 7]
```diff
@@ -33,6 +35,10 @@
 | secureTextEntry      |           | boolean                |              |
 | onChangeText         |           | (e) => {}              |              |
 | onSubmitEditing      |           | func                   |              |
+| leftIcon             |           | `ReactElement`         |              |
```
Comment (hyochan) [inline src/components/shared/EditText/README.md]: leftElement?

**Review by hyochan — APPROVED** [review]
👏 👏 👏 👏 👏 👏

**Pull request merged**